content/docs/1.4.0/references/settings.md

```diff
@@ -552,10 +548,14 @@ See [Trim Filesystem](../../volumes-and-nodes/trim-filesystem) for details.
 
 > Default:
 
-This deprecated setting is replaced by 2 new settings "Guaranteed Engine Manager CPU" and "Guaranteed Replica Manager CPU" since Longhorn version v1.1.1.
+This deprecated setting is replaced by 2 new settings `Guaranteed Engine Manager CPU` and `Guaranteed Replica Manager CPU` since Longhorn version v1.1.1.
```

Comment:
This is not deprecated but already removed instead. Please just remove this.

Reply:
Remove this description and the related bookmark item,